lib/logstash/outputs/opensearch/http_client.rb

```diff
@@ -385,12 +394,14 @@ def template_exists?(name)
 
     def template_put(name, template)
       path = "#{template_endpoint}/#{name}"
-      logger.info("Installing Elasticsearch template", name: name)
+      logger.info("Installing OpenSearch template", name: name)
       @pool.put(path, nil, LogStash::Json.dump(template))
     end
 
     def template_endpoint
-      maximum_seen_major_version < 8 ? '_template' : '_index_template'
+      # TODO: Check Version < 7.8 and use index template for >= 7.8 & OpenSearch
```

Comment:
Why cant we do this now?

Reply:
Sure. I created a ticket here: https://github.com/opensearch-project/logstash-output-opensearch/issues/12 for discussion. Can you comment on ticket about what do you think there?